flink-connectors/flink-connector-pulsar/src/main/java/org/apache/flink/connector/pulsar/common/schema/PulsarSchemaTypeSerializer.java

```diff
@@ -39,6 +39,7 @@
 import static org.apache.flink.util.Preconditions.checkState;
 
 /** Wrap the pulsar {@code Schema} into a flink {@code TypeSerializer}. */
+// TODO why do we have to use pulsar schema to serialize the data ?
```

Comment:
remove : Protobuf class can only be serialized using protobuf serializer. We can't serialize a Protobuf class using AVRO